Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMeraki.yaml

```diff
@@ -1,46 +1,580 @@
+                            replace_string(tostring(DstMacAddr), "'", "")
+                        )
+        | extend SrcIpAddr = iff(SrcIpAddr == SrcMacAddr, "", SrcIpAddr)
+        | extend DstIpAddr = iff(DstIpAddr == DstMacAddr, "", DstIpAddr)
+        | extend DvcMacAddr = column_ifexists("wired_mac", "")
+        | extend DvcMacAddr = iff(
+                            DvcMacAddr has '"',
+                            replace_string(tostring(DvcMacAddr), '"', ''),
+                            replace_string(tostring(DvcMacAddr), "'", "")
+                        )
+        | parse kind=regex Substring with * " VLAN " VLAN "[ ]{0,1}" restmessage
+        | parse kind=regex VLAN with VLAN2 " with " restmessage2
+        | parse kind=regex Substring with * "message: " message "[ ]{0,1}" restmessage
+        | extend
+            DvcAction = case(
+                    Substring has "allow",
+                    "Allow",
+                    Substring has "deny",
+                    "Deny",
+                    Substring has "Blocked",
+                    "Deny",
+                    ""
+                ),
+            EventMessage = iff(
+                    LogSubType has_any("Blocked DHCP server", "Virtual router collision"),
+                    Substring,
+                    coalesce(message, "")
+                ),
+            SrcUserName = column_ifexists("identity", ""),
+            SrcVlanId = coalesce(VLAN2, VLAN, vlan_id)
+        | extend
+            direction= iff(
+                direction has '"',
+                replace_string(tostring(direction), '"', ''),
+                replace_string(tostring(direction), "'", "")
+            ),
+            priority = iff(
+                priority has '"',
+                replace_string(tostring(priority), '"', ''),
+                replace_string(tostring(priority), "'", "")
+            ),
+            protocol = iff(
+                protocol has '"',
+                replace_string(tostring(protocol), '"', ''),
+                replace_string(tostring(protocol), "'", "")
+            ),
+            duration = iff(
+                duration has '"',
+                replace_string(tostring(duration), '"', ''),
+                replace_string(tostring(duration), "'", "")
+            ),
+            reason = iff(
+                reason has '"',
+                replace_string(tostring(reason), '"', ''),
+                replace_string(tostring(reason), "'", "")
+            ),
+            SrcVlanId =  iff(
+                    SrcVlanId has '"',
+                    replace_string(tostring(SrcVlanId), '"', ''),
+                    replace_string(tostring(SrcVlanId), "'", "")
+                ),
+            SrcUserName =  iff(
+                    SrcUserName has '"',
+                    replace_string(tostring(SrcUserName), '"', ''),
+                    replace_string(tostring(SrcUserName), "'", "")
+                )
```

Comment:
Line 426 to 460 is not required if quotes are used in parse-kv.

Reply:
Done